src/main/java/org/candlepin/subscriptions/capacity/admin/InternalSubscriptionResource.java

```diff
@@ -83,14 +87,43 @@ public AwsUsageContext getAwsUsageContext(
             Usage.fromString(usage),
             BillingProvider.AWS,
             billingAccountId);
+
+    // Set start date one hour in past to pickup recently terminated subscriptions
+    var start = date.minusHours(1);
     List<Subscription> subscriptions =
         subscriptionSyncController.findSubscriptionsAndSyncIfNeeded(
-            accountNumber, Optional.empty(), usageKey, date, date, true);
+            accountNumber, Optional.empty(), usageKey, start, date, true);
+
+    var existsRecentlyTerminatedSubscription =
+        subscriptions.stream()
+            .anyMatch(
+                subscription ->
+                    subscription.getEndDate().isBefore(date)
+                        && !subscription.getEndDate().isEqual(date));
```

Comment:
nitpick: unneeded second clause (always `true`)
```suggestion
                    subscription.getEndDate().isBefore(date);
```